preamble.tex

```diff
@@ -57,7 +57,7 @@
 \fi
 \setcounter{totalnumber}{1000}
 \raggedbottom
-\usepackage{hyperref}
+\usepackage[bookmarksnumbered=true]{hyperref}
```

Comment:
Wouldn't it be better to place the options 2 lines below as part of the hypersetup. Seems cleaner that way.

Reply:
Yes. I moved it.